core/model/src/main/java/org/eclipse/rdf4j/model/util/RDFContainers.java

```diff
@@ -0,0 +1,65 @@
+/*******************************************************************************
+ * Copyright (c) 2020 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ *******************************************************************************/
+
+package org.eclipse.rdf4j.model.util;
+
+import org.eclipse.rdf4j.model.*;
+import org.eclipse.rdf4j.model.impl.SimpleValueFactory;
+import org.eclipse.rdf4j.model.vocabulary.RDF;
+import org.eclipse.rdf4j.model.vocabulary.RDFS;
+
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Iterator;
+import java.util.Objects;
+import java.util.function.Consumer;
+
+/**
+ * TODO add documentation
+ */
+public class RDFContainers {
+
+	public static <C extends Collection<Statement>> C asRDF(IRI containerType, Iterable<?> values, Resource container, C sink,
```

Comment:
Perhaps a nice addition would be to also have conversion methods that pick the container type, e.g `asAlt`, `asSeq`, and `asBag`.

Reply:
Actually the `containerType` passed into the `asRDF` method is for that purpose only. We will use the `containerType` parameter in further implementations. Should we have those separate methods as well?

Reply:
Just a comment on the use of “as”. Java has “toString” and that’s made me use “to” rather than “as” in other code.

Reply:
@hmottestad I think in this case it's modeled off of the naming we use in the RDFCollections class for similar methods. I kind of agree with you that 'to' might have been a better choice ('as' to me suggests a view on an object rather than a new, converted, object), but to stay consistent I'm fine with sticking with this.

Reply:
@hmottestad  actually, coming back to it, I think you are right, and we might as well change this. @reeshabhkumarranjan do you think you could do a fix to change all the `as...` method names `asRDF`, `asValues` to `to...` method names (`toRDF`, `toValues`, etc.)?

Reply:
@jeenbroekstra Yes I can do this. Just to confirm, I have to do this only in `RDFContainers.java`, right?

Reply:
Yes, that's correct, only `RDFContainers`. We'll fix RDFCollections later, separately.